### PR TITLE
feat(@vtmn/react): add `VtmnPopover` component

### PR DESCRIPTION
**Pull request by lauthieb**
## Checklist

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [x] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [x] Check your code additions will fail neither code linting checks.
- [x] I have reviewed the submitted code.
- [x] I have tested on related showcases.
- [x] If it includes design changes, please ask for a review with a core team designer.

## Does this introduce a breaking change?

- No

### CODE REVIEW

File: packages/showcases/core/csf/components/overlays/popover.csf.js [hunk 1]
```diff
@@ -7,3 +7,39 @@ export const parameters = {
...
+      'top',
+      'top-right',
+      'right',
+      'bottom-right',
+      'bottom',
+      'bottom-left',
+      'left',
+    ],
+    description: 'The position of the popover.',
+    defaultValue: 'bottom',
```
Comment (GaspardMathon) [inline packages/showcases/core/csf/components/overlays/popover.csf.js:25]:
```suggestion
    defaultValue: 'top',
```

Isn't it top by default ?

File: packages/showcases/core/csf/components/overlays/popover.csf.js [hunk 2]
```diff
@@ -7,3 +7,39 @@ export const parameters = {
...
+  },
+  placeholder: {
+    type: { name: 'string', required: false },
+    description: 'The placeholder of the popover.',
+    defaultValue: 'Placeholder Text',
+    control: { type: 'text' },
+  },
```
Comment (GaspardMathon) [inline packages/showcases/core/csf/components/overlays/popover.csf.js:32]: What is a placeholder for the tooltip ?

File: packages/sources/react/src/components/overlays/VtmnPopover/VtmnPopover.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+}: VtmnPopoverProps) => {
+  return (
+    <div
+      className="vtmn-popover"
+      data-position={position}
+      aria-describedby={identifier}
+      tabIndex={0}
+      {...props}
+    >
+      <VtmnLink>{placeholder}</VtmnLink>
```
Comment (GaspardMathon) [inline packages/sources/react/src/components/overlays/VtmnPopover/VtmnPopover.tsx:57]: Ok, I understand for the placeholder, for me it should not be named "placeholder" and I don't think we should show a VtmnLink to highlight the text, what do you think ?

**Review by GaspardMathon — CHANGES_REQUESTED** [review]

**Comment by lauthieb** [comment]
@GaspardMathon everything is fixed, I merge :)

**Pull request merged**